test/e2e/client_tls_test.go

```diff
@@ -1441,7 +1385,7 @@ func verifyCRLs(t *testing.T, pod *corev1.Pod, expectedCRLs map[string]*x509.Rev
 	if err != nil {
 		return false, err
 	}
-	if len(activeCRLs) == 0 {
+	if len(activeCRLs) == 0 && len(expectedCRLs) != 0 {
```

Comment:
Was this change necessary for any of the test cases?

Reply:
Looks like it's not necessary. Reverted.